src/worker/tasks/sendTransactionWorker.ts

```diff
@@ -155,7 +168,9 @@ const _sendTransaction = async (
         client: thirdwebClient,
         chain,
         ...queuedTransaction,
-        // Stub the nonce because it will be overridden later.
+        // Unset gas overrides to estimate onchain values.
+        maxFeePerGas: undefined,
```

Comment:
By unsetting any overrides, this line will always estimate maxFeePerGas from RPC.